src/wallet/rpcwallet.cpp

```diff
@@ -3967,7 +3967,8 @@ UniValue z_getmigrationstatus(const UniValue& params, bool fHelp) {
     {
         std::vector<CSproutNotePlaintextEntry> sproutEntries;
         std::vector<SaplingNoteEntry> saplingEntries;
-        pwalletMain->GetFilteredNotes(sproutEntries, saplingEntries, "", 1);
+        std::set<PaymentAddress> zaddrs;
```

Comment:
I suggest renaming this to `noFilter`.

Reply:
Also add a comment saying what this is intending to do. If I'm not mistaken, it's intended to find unspent notes sent in transactions at any chain depth or in the mempool, to any address for which we have a spending key, including locked notes.